xonsh/tools.py

```diff
@@ -228,6 +228,30 @@ def __radd__(self, other):
             other = other._l
         return EnvPath(other + self._l)
 
+    def add(self, data, toStart=False, replace=False):
```

Comment:
PEP8: Python variable names should be `lower_case_with_underscores`. Also, may I recommend that `start` be renamed to `beg`, `begining`, or `front`? Thanks!